skt/executable.py

```diff
@@ -203,15 +218,15 @@ def cmd_merge(cfg):
     merge_report_path = join_with_slash(cfg.get('output_dir'),
                                         'merge.report')
 
-    try:
-        if cfg.get('merge_ref'):
-            report_string = '\n{}\n{}'.format(
-                report_string,
-                'We merged the following references into the tree:'
-            )
-            for mb in cfg.get('merge_ref'):
-                save_state(cfg, {'mergerepo_%02d' % idx: mb[0],
-                                 'mergehead_%02d' % idx: bhead})
+    for thing_to_merge in cfg.get('merge_queue', []):
+        try:
+            if thing_to_merge[0] == 'merge_ref':
+                if not previous_merge[0] == 'merge_ref':
```

Comment:
Can we use a less confusing syntax of `if previous_merge[0] != 'merge_ref':` ?

Reply:
I fix it.